kubernetes/ama-logs.yaml

```diff
@@ -394,8 +394,8 @@ spec:
           imagePullPolicy: IfNotPresent
           resources:
             limits:
-              cpu: 500m
-              memory: 750Mi
+              cpu: 3
```

Comment:
Did we update this in aks-rp manifest?

Reply:
> Did we update this in aks-rp manifest?

I will be making  this as part of 3.1.21 release.